templates/base/navigation.html

```diff
@@ -1,51 +1,117 @@
+        <strong class="govuk-tag govuk-tag--yellow govuk-phase-banner__content__tag">{{ ENV|title }}</strong>
+      {% endif %}
+      {% if ENV == "test" %}
+        <strong class="govuk-tag govuk-tag--green govuk-phase-banner__content__tag">{{ ENV|title }}</strong>
+      {% endif %}
+      {% if ENV == "preprod" %}
+        <strong class="govuk-tag govuk-tag--blue govuk-phase-banner__content__tag">{{ ENV|title }}</strong>
+      {% endif %}
+  <!--<span class="govuk-phase-banner__text">
+    This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.
+  </span>-->
+      <span class="govuk-phase-banner__text">
+        This service is currently in development and feedback is being gathered through user research.
+      </span>
+    </p>
+    </div>
+  </div>
+</div>
+
+
+<div class="moj-primary-navigation">
+
+  <div class="moj-primary-navigation__container">
+
+    <div class="moj-primary-navigation__nav">
+
+      <nav class="moj-primary-navigation" aria-label="Primary navigation">
+
+        <ul class="moj-primary-navigation__list">
           {% url 'home:home' as home_url %}
           {% url 'home:search' as search_url %}
           {% url 'home:glossary' as glossary_url %}
-          <li class="govuk-header__navigation-item {% if request.path == home_url %}govuk-header__navigation-item--active{%endif%}">
-            <a class="govuk-header__link" href="{% url 'home:home' %}">
+          <li class="moj-primary-navigation__item">
+            <a class="moj-primary-navigation__link" href="{% url 'home:home' %}" {% if request.path == home_url %}aria-current="page"{% endif %}>
               Home
             </a>
           </li>
-          <li class="govuk-header__navigation-item {% if request.path == search_url %}govuk-header__navigation-item--active{%endif%}">
-            <a class="govuk-header__link" href="{% url 'home:search' %}?new=True">
+          <li class="moj-primary-navigation__item">
+            <a class="moj-primary-navigation__link" href="{% url 'home:search' %}?new=True" {% if request.path == search_url %}aria-current="page"{% endif %}>
               Search
             </a>
           </li>
-          <li class="govuk-header__navigation-item {% if request.path == glossary_url %}govuk-header__navigation-item--active{%endif%}">
-            <a class="govuk-header__link" href="{% url 'home:glossary' %}">
+          <li class="moj-primary-navigation__item">
+            <a class="moj-primary-navigation__link" href="{% url 'home:glossary' %}" {% if request.path == glossary_url %}aria-current="page"{% endif %}>
               Glossary
             </a>
           </li>
+
         </ul>
-        <p class="govuk-body govuk-!-text-align-right"><a class="govuk-link govuk-link--inverse govuk-!-font-weight-bold govuk-link--no-underline" href="{% url 'azure_auth:logout' %}">Sign Out</a>
-        </p>
+
       </nav>
+
     </div>
+    <!-- <div class="moj-primary-navigation__search">
```

Comment:
Is this meant to be commented out?

Reply:
Nope, this can be removed actually. I'll pick it up next week 👍🏻